govcd/openapi_org_network_test.go

```diff
@@ -418,6 +418,19 @@ func nsxtRoutedDhcpConfig(check *C, vdc *Vdc, orgNetId string) {
 	err = vdc.DeleteOpenApiOrgVdcNetworkDhcp(orgNetId)
 	check.Assert(err, IsNil)
 
+	orgVdcNetwork, err := vcd.org.GetOpenApiOrgVdcNetworkById(orgNetId)
+	check.Assert(err, IsNil)
+	check.Assert(orgVdcNetwork, NotNil)
+
+	updatedDhcp2, err := orgVdcNetwork.UpdateDhcp(dhcpDefinition)
+	check.Assert(err, IsNil)
+	check.Assert(updatedDhcp2, NotNil)
+
+	check.Assert(dhcpDefinition, DeepEquals, updatedDhcp2.OpenApiOrgVdcNetworkDhcp)
+
+	err = orgVdcNetwork.DeletNetworkDhcp()
+	check.Assert(err, IsNil)
```

Comment:
we could verify that doesn't exist anymore:) 
`orgVdcNetwork, err := vcd.org.GetOpenApiOrgVdcNetworkById(orgNetId)`

Reply:
resolved